### ISSUE TITLE
Keep Evernote reminders by exposing them to templates

### ISSUE THREAD

**Issue by xmacex** [akosbalasko/yarle#348]
Hi, this is very powerful thanks.

I wonder if the Evernote reminder items could please be exposed to be included in templates.

They look like this in ENEX files:

```xml
<note>
    <note-attributes>
      <reminder-time>20181119T082700Z</reminder-time>
      <reminder-done-time>20181121T230739Z</reminder-done-time>
      <reminder-order>1542616020000</reminder-order>
    </note-attributes>
    ⋮
</note>
```

I would love to snatch those from the thousands of existing notes I have, and have them as something like

```yaml
---
reminder-time: 2018-11-19 08:27
reminder-done-time: 2018-11-21 23:07
reminder-order: 1542616020000
---
```

On Evernote it is possible to have a reminder which has no `reminder-time` set, and these are handy "do this sometimes" todo items. Such notes have a `reminder-order` which indicates that there is *some* reminder on them, but don't have a `reminder-date`.

I looked at `yarle/utils/templates/placeholders` and `yarle/utils/templates/apply-functions` directories, but then got lost, unfortunately as I'm not so familiar with modern JavaScript and TS.

For context, I'm one of these people on Evernote → Obsidian exodus. Discovering Yarle gives me hope 🤞

**Comment by xmacex** [comment]
Hello, nice to hear if this feature would be interesting to other users and you devs too. I have found over the years that the "one-note-per-todo-action" is a good paradigm, and at least one which works well for me. Rich annotations, attachments, emails, sketches, PDFs and possible task lists too are available, but not necessary in this style of doing things. It worked well with Evernote until recently, but you know...

I surveyed Yarle code and my intuition would be to basically adopt and adapt one of the other metadata properties in the *MetaData* model and , it's extractor function, the template placeholders, processor function and add template config, and docs (phew many things to keep track of without familiarity with the codebase, I'm sure I'm missing something), but I didn't manage to get a dev setup going unfortunately. What I'm trying to say it looks straightforward thanks to the well but existing infrastructure, basically just extending what Yarle already does to three new fields in the Evernote data model.

Cheers :)
